content.test.ts

```diff
@@ -0,0 +1,51 @@
+import fs from 'fs';
+import path from 'path';
+
+describe('Codecademy Docs Content', () => {
+  it('adheres to content file structure', () => {
+    // file names can only contain alphanumerics and hyphens
+    const validateName = (pathName: string, name: string) => {
+      const filteredName = name.replace(/[^A-Za-z0-9.-]/g, '');
+
+      // format so that test failures are more helpful
+      if (filteredName !== name) {
+        expect(`${pathName} - file name must only include alphanumerics or -`).toBe('');
+      }
+    };
+
+    // nodes can have n directories of children and at most one .md file with the same name as its parent directory
```

Comment:
It is possible that this will change to specify _exactly one_ .md file once the content is ready for that